### PR TITLE
tp-qemu: Add case WMI_facility_test for windows guests

### PR DESCRIPTION
**Pull request by peixiu**
step1: boot guest vm up with num_queues
step2: enable vioscsi.sys verifier in guest
step3: run wmi check command
step4: check instance.QueuesCount value as expected

id: 1564837
Signed-off-by: Peixiu Hou <phou@redhat.com>

### CODE REVIEW

**Comment by peixiu** [comment]
(1/1) Host_RHEL..u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_mq.WMI_facility_check.q35: PASS (162.17 s)

**Comment by kimi1978wy** [comment]
Acked-by: Yu Wang <wyu@redhat.com>

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

File: qemu/tests/cfg/virtio_scsi_mq.cfg [hunk 1]
```diff
@@ -14,3 +13,14 @@
...
+    variants:
+        - @default:
+            only RHEL
+        - WMI_facility_check:
+            only Windows
+            # As WinXP, Win2003, Win2008, Win7 not support multi queue. So don't test them.
+            no WinXP Win2003 Win2008 Win7
+            system_image_drive_format = scsi-hd
+            driver_name = vioscsi
+            pattern = "instance.QueuesCount=(\d+)"
```
[the remark this answers is not in the document]
Comment (YongxueHong) [reply qemu/tests/cfg/virtio_scsi_mq.cfg]: my bad,  the above suggestion is "instance\.QueuesCount=(\d+)", sorry.

[the remark this answers is not in the document]
Comment (peixiu) [reply qemu/tests/cfg/virtio_scsi_mq.cfg]: Updated, thanks~

File: qemu/tests/cfg/virtio_scsi_mq.cfg [hunk 2]
```diff
@@ -14,3 +13,14 @@
...
         system_image_drive_format = scsi-hd
         scsi_hba_image1 = spapr-vscsi
+    variants:
+        - @default:
+            only RHEL
+        - WMI_facility_check:
+            only Windows
+            # As WinXP, Win2003, Win2008, Win7 not support multi queue. So don't test them.
+            no WinXP Win2003 Win2008 Win7
+            system_image_drive_format = scsi-hd
```
[the remark this answers is not in the document]
Comment (peixiu) [reply qemu/tests/cfg/virtio_scsi_mq.cfg]: Because avocado used  default system image drive format is scsi-hd, so here keep consistency with default used.

**Comment by peixiu** [comment]
Tested with latest code, it can be passed.
  (1/1) Host_RHEL.m*.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_mq.WMI_facility_check: PASS (138.40 s)

**Comment by peixiu** [comment]
(1/1) Host_RHEL.m*.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_mq.wmi_facility_check: PASS (82.94 s)

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by xuwei@redhat.com

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**